package-lock.json

```diff
@@ -0,0 +1,1953 @@
+{
```

Comment:
package-lock.json needed?

Reply:
It's generally good practice to commit these as they lock in packages at versions you know they're working at. It also helps you know when packages have changed since it may not be apparent when you make updates.

Reply:
Oh, I forgot the most important reason for these -- they make `npm install` **fast**.

Reply:
Okay great. Just wanted to know.